reporters_db/data/reporters.json

```diff
@@ -13519,13 +13561,16 @@
                 "1 N.C. 1",
                 "46 N.C. (1 Jones) 456",
                 "61 N.C. (Phil.) 456",
-                "20 N.C. (3 & 4 Dev. & Bat.) 456"
+                "20 N.C. (3 & 4 Dev. & Bat.) 456",
```

Comment:
I think works because everything is hardcoded in regex in [L13554](https://github.com/freelawproject/reporters-db/blob/4754ef5b65e8cead851400ddc8c6e789da7ae2df/reporters_db/data/reporters.json#L13555)